frontend/public/components/overview/pods-overview.tsx

```diff
@@ -0,0 +1,78 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { ListGroup } from 'patternfly-react';
+
+import { ResourceLink, resourcePath, SidebarSectionHeading, StatusIconAndText } from '../utils';
+import {podPhase, PodKind, K8sResourceKind} from '../../module/k8s';
```

Comment:
nit: format this file with prettier since it's a new file and not part of the `packages` dir where the newer eslint rules apply.